src/QMCDrivers/QMCDriverNew.cpp

```diff
@@ -221,18 +223,17 @@ void QMCDriverNew::putWalkers(std::vector<xmlNodePtr>& wset)
     return;
   const int nfile = wset.size();
 
-  auto& W = population_.getWalkerConfigsRef();
-  HDFWalkerInputManager W_in(W, population_.get_golden_electrons()->getTotalNum(), myComm);
+  HDFWalkerInputManager W_in(walker_configs_ref_, population_.get_golden_electrons().getTotalNum(), myComm);
```

Comment:
I have no idea about qmc_common.is_restart. Likely it can be deleted.

I also don't like loading walker configurations from h5 inside a driver. I would prefer also the mcwalkerset XML tag to be parsed directly rather then inside a driver. The configurations directly goes into the main state WalkerConfigurations.

I think the old way stores multiple mcwalkerset lines and a specific driver decides how to do use them maybe one maybe more.

Reply:
Done in #4140